go/interpreter/lfvm/instructions_test.go

```diff
@@ -733,7 +733,7 @@ func TestTransientStorageOperations(t *testing.T) {
 		setup    func(*tosca.MockRunContext)
 		stackPtr int
 		revision tosca.Revision
-		status   Status
+		status   status
```

Comment:
nit: field name same as type name is something I think should be avoided, but I guess in tests it is not such a terrible thing